src/main/java/org/folio/event/service/UserTenantService.java

```diff
@@ -129,4 +144,13 @@ private Future<Void> isUsernameUniqueAcrossTenants(String username, String conso
         return Future.succeededFuture();
       }));
   }
+
+  private Future<Void> isUserTypePopulated(User user) {
+    if (Objects.isNull(user.getType())) {
```

Comment:
Thinking lets make check more strict, so fo consortia mode lets check that type equals to some of this values: shadow, staff, patron. So we also update error message with possible values.
Also the same validation we have on UI